pkg/service/export.go

```diff
@@ -54,14 +55,34 @@ func Export(clientProducer client.Producer, serviceNamespace, svcName string, st
 		Create(context.TODO(), resourceServiceExport, metav1.CreateOptions{})
 	if err != nil {
 		if k8serrors.IsAlreadyExists(err) {
-			status.Success("Service already exported")
+			status.Success(fmt.Sprintf("Service %s/%s already exported", serviceNamespace, svcName))
 			return nil
 		}
 
-		return status.Error(err, "Failed to export Service")
+		return status.Error(err, fmt.Sprintf("Failed to export Service %s/%s", serviceNamespace, svcName))
 	}
 
-	status.Success("Service exported successfully")
+	status.Success(fmt.Sprintf("Service %s/%s exported successfully", serviceNamespace, svcName))
+
+	return nil
+}
+
+func Exports(clientProducer client.Producer, serviceNamespace string, status reporter.Interface) error {
+	svcs, err := clientProducer.ForKubernetes().CoreV1().Services(serviceNamespace).List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		return status.Error(err, "Unable to list the Services in namespace %q", serviceNamespace)
+	}
+
+	if len(svcs.Items) == 0 {
+		status.Warning("No Services exist in target namespace")
+		return nil
+	}
+
+	for _, svc := range svcs.Items {
+		if err := Export(clientProducer, svc.Namespace, svc.Name, status); err != nil {
```

Comment:
I wonder what behaviour would be more useful:

* stop at the first error (as happens now)
* try to export all services in the namespace, accumulating errors

What do you think?